pymyenergi/client.py

```diff
@@ -248,6 +247,11 @@ async def refresh(self):
                         f"Updating {existing_device.kind} {existing_device.name}"
                     )
                     existing_device.data = device_data
+
+                # Update the extra information available on libbi
+                # this is the bit that requires OAuth
+                if existing_device.kind == LIBBI:
```

Comment:
Perhaps add a check here to see if app email and password is set and print an error if not?